README.md

```diff
@@ -40,6 +40,24 @@ You can try out the demo instance: http://demo.redash.io/ (login with any Google
 * Slack: http://slack.redash.io/
 * Gitter (chat): https://gitter.im/getredash/redash
 
+## Running Tests
+
+```
+$ export PYTEST_ENABLE_REDIS_CLEANUP=False
+$ export REDASH_REDIS_URL="redis://localhost:6379/0"
+$ export REDASH_DATABASE_URL="postgresql://postgres@localhost/postgres"
+$ docker-compose run -d --service-ports redis
+$ docker-compose run -d --service-ports postgres
+$ pip install -U virtualenv setuptools
+$ virtualenv venv && . venv/bin/activate
+$ pip install -r requirements_dev.txt
+$ pip install -r requirements.txt
+$ npm install
+$ npm run build
+$ pytest tests
+```
```

Comment:
This doesn't belong in the README, but in the developer docs. Also there is a much simpler method to run the tests (see `Running Tests` [here](https://redash.io/help-onpremise/dev/docker.html)).